conf/test_github_full.config

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I can't see this profile being used in any Github workflow

Reply:
should change back to the non paid server

Reply:
Changed the CI to use the full profile.

The local testing yamls are so we all use the same yaml for test runs, we haven't been using these as part of any profile although I could set them up as such.

Reply:
The BGA.yaml is specific to the gitpod that is included in the repo.